internal/controllers/cryostat_controller_test.go

```diff
@@ -2152,8 +2228,9 @@ func (t *cryostatTestInput) checkReportsDeployment() {
 	}
 
 	checkReportsContainer(&template.Spec.Containers[0], t.TLS, t.EnvReportsImageTag, resources, test.NewReportSecurityContext(cr))
-	// Check that the proper Service Account is set
-	Expect(template.Spec.ServiceAccountName).To(Equal("cryostat"))
+	// Check that the default Service Account is used
+	Expect(template.Spec.ServiceAccountName).To(BeEmpty())
```

Comment:
This is only specific to `FakeClient`, right? In actual cluster, it would be set to `default`?

Reply:
This is for the real cluster too. When the `serviceAccountName` is left empty, the default service account is used: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#use-the-default-service-account-to-access-the-api-server

Reply:
Ahh right! I just realized golang string default-value is empty. Thanks for explaining that :D